### DISCUSSION TITLE
Show entries tree list collapsed by default

### DISCUSSION THREAD

**Discussion by eduardoalonsoalbella** [craftcms/cms#12186] [Ideas] [answered]
Hi, once we start having lots of entries in craft, we try to organize everything following a tree structure, the problem is that showing the entries list expanded by default, makes it difficult to navigate thourought the tree because, you need first to collapse everything and then navigate thourought the nested items.

It would be better if the tree was collapsed by default instead of expanded.

In our case, we hace 15 first levels, and each sublevel has maybe more than 1000 articles with more nested levels, making navigation quite tedious.

**Comment by brandonkelly** [comment, marked as answer]
I don’t think we will change the default, but we are considering adding “Collapse All” and “Expand All” buttons (#11531).

**Reply by eduardoalonsoalbella** [reply]
@brandonkelly do you an ETA for this? Thanks

**Reply by brandonkelly** [reply]
@eduardoalonsoalbella Not yet because it would likely be a big performance hit, the way things work right now, so will require a little refactoring. We’re thinking it through though.

**Comment by eduardoalonsoalbella** [comment]
That would be great!
